### DISCUSSION TITLE
SYNC OF CLIPPED CONTENT ACROSS PCs

### DISCUSSION THREAD

**Discussion by qalisto** [hluk/CopyQ#1753] [Q&A]
I have enabled sync to one, cloud folder on all my CPQ clients.

How do I enable all the machines to show the same Clips / Tabs from that Sync'd folder?

I assume this is possible.  But I don't see how to set up the client to do this.

Please kindly advise.

**Comment by hluk** [comment]
Please see the documentation: https://copyq.readthedocs.io/en/latest/synchronize.html

**Reply by qalisto** [reply]
Sorry, but I am using Windows and I had already directed the Sync to the Folder desired.

I surmise that this is incomplete and I need to find and install the "synch-" plugin

BUT
As a newbie, I do not see any obvious way to locate:

(a) the repository of plugins, like Synchronize  AND THEN

(b) where / how to install such a plugin *after* I find it.

I see no plugins folder nor do I see a catalog of plugins that appears in search.  I see only a list of plugins which are in Code and do not appear to be downloadable files to be installed like other plugin-capable applications.

What is my step by step for Windows?

Apologies if I missed something.

**Reply by hluk** [reply]
Plugins can be selected during installation of the app on Windows, and also included in the portable zip file. I will make it clearer in the docs.

**Reply by qalisto** [reply]
Does one then ... reinstall?  copy paste the Plug-In into the Plug-In folder?  Yes, suggest that Plug-Ins deserve a section and description of their respective function and ability to download each, discrete plugin for future use.  Thanks!
